metadata/src/utils/validation.rs

```diff
@@ -503,6 +534,7 @@ impl<'a> MetadataHasher<'a> {
             metadata,
             specific_pallets: None,
             specific_runtime_apis: None,
+            include_custom_values: true,
```

Comment:
Cool!

Let's add an `pub fn ignore_custom_values(&mut self) -> &mut Self { self.include_custom_values = false; self }` maybe as well so that people can opt out of hashing custom values :)

Reply:
This also still should be added